doc/users/next_whats_new/set_colors_ticks_and_ticklabels_rcparams.rst

```diff
@@ -0,0 +1,34 @@
+The color of ticks and tick labels can be set independently from rcParams
+-------------------------------------------------------------------------
+
+The color of ticks and tick labels can now be set independently from the
+matplotlib rc file or the rcParams using the :rc:`xtick.color` or 
+:rc:`ytick.color` parameters to set the tick colors and the 
+:rc:`xtick.labelcolor` or :rc:`ytick.labelcolor` parameters to set the tick 
```

Comment:
More concise:
~~~
:rc:`xtick.color` used to define the tick color and the label color. The label
color can now be set independently using :rc:`xtick.labelcolor`. It defaults
to "inherit" which will take the value from :rc:`xtick.color`. The same holds
for ``ytick.[label]color``.
~~~

Reply:
Thank you @timhoffm I've implemented your suggestions.